daemon/daemon_unix.go

```diff
@@ -186,7 +190,10 @@ func parseSecurityOpt(container *container.Container, config *containertypes.Hos
 
 		switch con[0] {
 		case "label":
-			labelOpts = append(labelOpts, con[1])
+			if selinuxConvert {
+				opt = strings.Replace(opt, ":", "=", 1)
```

Comment:
I mean, this is for the old colon format for security options:

label:type:scp_t

That must be passed down to init labels as

label=type:scp_t